packages/react-components/src/components/VideoTile.tsx

```diff
@@ -341,11 +349,11 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
           background: theme.palette.neutralLighter,
           borderRadius: theme.effects.roundedCorner4
         },
-        isSpeaking && {
+        (isSpeaking || /* @conditional-compile-remove(raise-hand) */ raisedHand) && {
           '&::after': {
             content: `''`,
             position: 'absolute',
-            border: `0.25rem solid ${theme.palette.themePrimary}`,
+            border: `0.25rem solid ${isSpeaking ? theme.palette.themePrimary : theme.palette.orangeLighter}`,
```

Comment:
this color looks incorrect in your test app

| testapp | figma |
| -- | -- |
| ![image](https://github.com/Azure/communication-ui-library/assets/2684369/52690e46-3f61-4c1f-ad0b-e40370e73bef) | ![image](https://github.com/Azure/communication-ui-library/assets/2684369/b39e8b01-1168-470e-a259-c763e261c2d4) |

There likely isn't an existing theme slot for this color so we will need to add a new semantic slot to the theme provider similar to how we have done for the end call button. I think we should add something like: `raiseHandGold` to this 
![image](https://github.com/Azure/communication-ui-library/assets/2684369/100433f9-c211-4de4-8757-5058d2b7ed80)